bundles/org.openhab.ui/doc/components/oh-button.md

```diff
@@ -99,11 +99,21 @@ Button performing an action
     Name of the variable to set on input change
   </PropDescription>
 </PropBlock>
+<PropBlock type="TEXT" name="variableKey" label="Variable Key">
+  <PropDescription>
+    Consider the variable value is an object and set the corresponding deep property within that object using a key syntax. Examples: <ul><li><code>user.name</code></li><li><code>user[0].address[1].street</code></li><li><code>[0]</code></li><li><code>[0].label</code></li></ul>The inner property and its parent hierarchy will be created if missing.
+  </PropDescription>
+</PropBlock>
 <PropBlock type="BOOLEAN" name="clearVariable" label="Clear Variable After Action">
   <PropDescription>
     Name of the variable to clear after performing the action
   </PropDescription>
 </PropBlock>
+<PropBlock type="BOOLEAN" name="clearVariable" label="Clear Variable After Action">
```

Comment:
This doesn't seem right. Did you regenerate the docs using the provided script?

See https://github.com/openhab/openhab-webui/blob/main/bundles/org.openhab.ui/CONTRIBUTING.md#documentation--resources
(admittedly it's a little hidden)